aea/registries/base.py

```diff
@@ -531,297 +521,3 @@ def fetch_internal_handler(self, skill_id: SkillId) -> Optional[Handler]:
         :return: the internal handler registered for the skill id
         """
         return self.fetch_by_protocol_and_skill(INTERNAL_PROTOCOL_ID, skill_id)
-
-
-class Resources:
```

Comment:
Resources is the only part of the public api in registries/* > removing it from here allows us to expose it in the api docs